Tools/Scripts/libraries/webkitbugspy/webkitbugspy/github.py

```diff
@@ -66,14 +66,26 @@ def from_string(self, string):
         return None
 
     def credentials(self, required=True):
```

Comment:
Is this method name (credentials) intentionally same as the one imported from webkitcorepy. Seems like that can create confusion.